apps/osie-installer.sh

```diff
@@ -11,6 +20,14 @@ fail() {
 		# before the timeout occurred.
 		if [ -f "${statedir}/autofail_stage" ]; then
 			stage=$(cat "${statedir}/autofail_stage")
+			# If SOL isn't working properly on deprovision, docker can hang even though
+			# the deprovision script has run to completion. Detect this by checking the
+			# autofail_stage value, which would indicate the deprovison completed. Don't
+			# report failure in that case, but just run the cleanup() function and exit.
+			if [ "$stage" = "deprovision completed" ]; then
```

Comment:
we shouldn't tie this to just deprovision, how about just handling `stage=completed`?